Assets/Scripts/Core/Match/PlayersController.cs

```diff
@@ -128,9 +128,12 @@ private void PassivelyCollectPrana()
                 {
                     if (player.Prana >= Constants.PranaValueMax) continue;
 
-                    var cities = _mapController.Cities
-                        .ByOwner(player.GUID)
-                        .SelectMany(city => !city.IsNeutral);
+                    var citiesByOwner = _mapController.Cities
+                        .ByOwner(player.GUID);
+                    var neutralCities = _mapController.Cities
+                        .SelectMany(city => city.IsNeutral)
+                        .SelectMany(city => city.GetComponent<NeutralStrategy>().IsPlayerCompetitor(player.GUID));
```

Comment:
Снова напоминаю про методы расширения `SelectMany` (см. CitiesExtensions в файле `Common/Extensions.cs`).
```suggestion
                        .SelectMany<NeutralStrategy>(city => city.IsPlayerCompetitor(player.GUID));
```